src/utils/workspace.py

```diff
@@ -24,29 +24,17 @@ def ws_auth(auth_token):
     return result['workspaces']
 
 
-def get_workspace_info(workspace_id, auth_token):
+def get_workspace_info(workspace_id, auth_token=None):
     """
     Given a list of workspace ids, return the associated workspace info for each one
     """
-    if not auth_token:
```

Comment:
I assume this will just return back an empty list right?

Reply:
Oops, my bad. This file has fixes that are not included. I just missed it while cherrypicking the selected changes  for this PR. I'll attend to it.